server/app/models/analysis.rb

```diff
@@ -95,10 +95,14 @@ class Analysis
   after_create :verify_uuid
   before_destroy :queue_delete_files
 
+  # TODO dynamically include as appropriate, depending on environment
+  include WebNode::Resque::Analysis
+
   def self.status_states
     %w(na init queued started completed)
   end
 
+  # FIXME analysis_type is somewhat ambiguous here, as it's argument to this method and also a class method name
```

Comment:
@nllong will you take a look at my second note from 2 weeks ago related to deleting?

Reply:
The delete would be the analysis directory from the os_server_project_path defined in the config.yml.  It should exist on your local computer, it would be in the server checkout under worker_nodes/server/...

https://github.com/NREL/OpenStudio-server/blob/develop/server/config/config.yml#L3